### PR TITLE
Show command buffer vulkan handle for subcommand node.

### PR DESCRIPTION
**Pull request by stellama0208**
- Related ticket: http://b/143734139.

### CODE REVIEW

File: gapis/resolve/command_tree.go [hunk 1]
```diff
@@ -512,6 +514,32 @@ func addFrameGroups(ctx context.Context, events *service.Events, p *path.Command
...
+func setSubCmdNodesName(
+	ctx context.Context,
+	r *api.SubCmdRoot,
+	refMap map[api.CmdID][]sync.SubcommandReference,
+	commands []api.Cmd) {
+
+	id := api.CmdID(r.Id[0])
+	refs := refMap[id]
+	r.TraverseSubCmdRoot(func(item *api.SubCmdRoot) {
+		switch len(item.Id) {
```
Comment (chrisforbes) [inline gapis/resolve/command_tree.go]: This seems a fairly hacky way of distinguishing the different types of nodes. Can we test for the actual conditions we want, or is that data gone by this point?

Comment (AWoloszyn) [reply gapis/resolve/command_tree.go]: This is not the right place to do this.
We should try and isolate the Vulkan-specific knowledge to the Vulkan package.

Comment (AWoloszyn) [reply gapis/resolve/command_tree.go]: You can see how we handle debug-marker regions as an example.

Comment (stellama0208) [reply gapis/resolve/command_tree.go]: Thanks for suggestions! Will learn how debug-marker regions do this for a while and may raise some questions during the process. Before finding a better solution will mark this PR as WIP for now.

**Comment by chrisforbes** [comment]
Superceded; Stella has landed a proper solution in AGI instead.

**Pull request closed without merging**